pkg/services/rediscache/catalog.go

```diff
@@ -3,6 +3,10 @@ package rediscache
 import "github.com/Azure/open-service-broker-azure/pkg/service"
 
 func (m *module) GetCatalog() (service.Catalog, error) {
+	bpd := newBasicPlanDetail()
```

Comment:
Can we just declare the planDetail structs inline here? This function isn't used anywhere else and I think it would be clearer if we just detailed it here so it was clear what was happening, without needing to jump to another file.

Reply:
The `planDetail` struct is also used when [generating plan schemas](https://github.com/Azure/open-service-broker-azure/pull/564/files#diff-b9a96ba77a9f65df70ee14de705dae70). Necessary information is extracted from the struct to generate plan schemas. If we make the struct inline, the function for generating plan schemas should also be inlined, which will makes this file very big.

Reply:
Is the `newBasicPlanDetail` function (or `newStandardPlanDetail()` or `newPremiumPlanDetail()` invoked anywhere but here? That was my point. Currently it looks like no, so you could just do this no?

```
bpd := planDetail{
   planName:        basic,
    allowedCapacity: []int64{0, 1, 2, 3, 4, 5, 6},
}
```

My comment was that the method is invoked only here, not in other places so it seemed unneeded.

Reply:
Aha, I get your point. Yes, this function is only used here. I'll change it later, thanks!